app/routes/course-admin/submissions.js

```diff
@@ -47,7 +56,9 @@ export default class CourseAdminSubmissionsRoute extends BaseRoute {
     return {
       course: course,
       languages: await this.store.findAll('language'),
+      course_stages: course.stages,
```

Comment:
Why do you need to pass `course.stages` explicitly when `course` is already passed? Also, this using snakecase instead of camelcase